package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "quick-build": "gulp quick-build",
     "build": "NODE_ENV=production gulp build",
-    "test": "gulp test",
-    "test:unit": "gulp test:unit",
+    "test": "TZ=UTC gulp test",
+    "test:unit": "TZ=UTC gulp test:unit",
```

Comment:
This looks unrelated. I remember the timezone being an issue for some tests a long time ago. I wonder if this could be better resolved by configuring the jest environment.

Either way, let's leave it out of this PR because it's unrelated.

Reply:
Ah right, this was something I asked early on in public channels and was told to just add it along with the change.

But I can confirm several date-related tests break if you do no run them on a machine in UTC. Unfortunately jest config doesn't have a good way of doing this (I remember facing the same problem multiple times), so the two typical options are to either set it in the script like this, or make sure to use the jest's `useFakeTimers` and `setSystemTime` in each test that checks dates.

Removed in the new revision. 👍